### ISSUE TITLE
API Key is currently hardcoded in forecast search

### ISSUE THREAD

**Issue by pantsesquire** [sudipt1999/Weather-report-app#8]
The secret.js file does not populate the api key in forecast.js. It is currently hardcoded.

**Comment by sudipt1999** [comment]
Yeah ! It's for just showcase project so it won't cause problem ! Well you can make a pr for this one and add steps to get API key in documentation!

**Comment by pantsesquire** [comment]
I've already added steps in another PR, I'll look into getting the secrets working

**Comment by pantsesquire** [comment]
#11
